### PR TITLE
Small grammar and punctuation suggestions

### PR DESCRIPTION
**Pull request by DanAtShenTech**

### CODE REVIEW

File: docs/introduction/the_architecture.md [hunk 1]
```diff
@@ -47,15 +47,15 @@ With a single state tree we can point to parts of the state using paths (the fir
...
+  1. There is no need for immutability in Cerebral because a change to a path means that any component depending on that path should render (no value comparison). In applications with large data structures, immutability has a high cost. There is no need to hack objects and arrays to observe changes to them either. There is nothing special about the state you put into Cerebral's state tree.
 
-  2. Since there is no value comparison in Cerebral it uses what we call **strict render**. This allows us to do render optimizations not possible with other solutions. For example you can say that a component depending on a list is only interested in added/removed items of the list or if the list itself is being replaced
+  2. Since there is no value comparison in Cerebral it uses what we call **strict render**. This allows us to do render optimizations not possible with other solutions. For example you can say that a component depending that depends on a list is only interested in added/removed items of the list or whether the list itself is being replaced.
```
Comment (henri-hulski) [inline docs/introduction/the_architecture.md]: "a component depending that depends on a list" sounds not really right ;)

**Comment by DanAtShenTech** [comment]
Checking to see if/when this pull request will be merged.

**Comment by henri-hulski** [comment]
We had to fix the next branch first, but now I can merge it.

**Comment by DanAtShenTech** [comment]
I see that my pull request was approved.  When do these updates show up on the live site?

**Comment by henri-hulski** [comment]
@DanAtShenTech this happens automatically on next release so that the website is always in sync with the latest release.

**Pull request merged**